app/javascript/components/Results/RelatedSearches/RelatedSearches.tsx

```diff
@@ -9,12 +13,19 @@ type RelatedSearch = {
 };
 
 interface RelatedSearchesProps {
+  affiliate: string;
   relatedSearches?: RelatedSearch[];
+  query: string;
+  vertical: string;
 }
 
-export const RelatedSearches = ({ relatedSearches=[] }: RelatedSearchesProps) => {
+export const RelatedSearches = ({ affiliate, relatedSearches=[], query, vertical }: RelatedSearchesProps) => {
   const i18n = useContext(LanguageContext);
 
+  const module = (() => {
+    return moduleCode.relatedSearches;
+  })();
```

Comment:
@lorawoodford there is a simpler way for this:
`const module = moduleCode.relatedSearches;`
I have done this on my latest PR https://github.com/GSA/search-gov/pull/1499 as per Steven's suggestion.